server/db/config/connection.js

```diff
@@ -14,8 +14,8 @@ switch (process.env.NODE_ENV) {
   case 'development':
     dbUrl = process.env.DEV_DB_URL;
     break;
-  default:
-    throw new Error('Database Url Not Found!');
+  // default:
+  //   throw new Error('Database Url Not Found!');
```

Comment:
if there is no database url , we should throw this error
```suggestion
    throw new Error('Database Url Not Found!');
```